Package.swift

```diff
@@ -3,7 +3,7 @@
 import PackageDescription
 
 let package = Package(
-    name: "PSPDFKit",
+    name: "Nutrient",
```

Comment:
In my testing, this is not a breaking change. I think this is just for display in the Name column here:

![Screenshot 2024-11-08 at 17 09 11](https://github.com/user-attachments/assets/96bd4bbe-c4ab-4596-959d-d42387a021d8)

Reply:
Yes, I agree, this seems safe to do.